loops-arrays-objects.js

```diff
@@ -32,9 +31,19 @@ const scores = [
 // D
 // ==========================================
 
-
-
-
+for (let i = 0; i < scores.length; i++) {
+    if (scores[i].score < 60) {
+        console.log(`F`);
+    } else if (scores[i].score >= 60 && scores[i].score < 70) {
```

Comment:
Hoi, deze for statement klopt maar het kan korter: (scores[i].score < 70), (scores[i].score < 80) etc. Door de volgorde van de loop hoef je niet groter dan 60 en kleiner dan 70 te schrijven maar slechts kleiner dan 70.